wikithingsdb/query.py

```diff
@@ -0,0 +1,203 @@
+import sys
+
+from peewee import fn
+
+from wikithingsdb.create import get_hypernyms
+from wikithingsdb.models import Article, Type, WikiClass, DbpediaClass,\
+    ArticleClass, ArticleType, Hypernym
+from wikithingsdb.util import to_wikipedia_class, to_dbpedia_class, \
+    from_dbpedia_class
+
+
+def types_of_article(article, limit=sys.maxint):
```

Comment:
`sys.maxint` is clever -- but is it hacky? I'm fine with this as long as it's not a frowned upon thing.